python/ct/crypto/asn1/oid.py

```diff
@@ -165,17 +165,24 @@ def _decode_value(cls, buf, strict=True):
 ID_DSA_WITH_SHA1 = ObjectIdentifier(value="1.2.840.10040.4.3")
 ID_EC_PUBLICKEY = ObjectIdentifier(value="1.2.840.10045.2.1")
 ECDSA_WITH_SHA1 = ObjectIdentifier(value="1.2.840.10045.4.1")
-# http://tools.ietf.org/html/rfc5758
+# RFC 5758
 ECDSA_WITH_SHA224 = ObjectIdentifier(value="1.2.840.10045.4.3.1")
 ECDSA_WITH_SHA256 = ObjectIdentifier(value="1.2.840.10045.4.3.2")
 ECDSA_WITH_SHA384 = ObjectIdentifier(value="1.2.840.10045.4.3.3")
 ECDSA_WITH_SHA512 = ObjectIdentifier(value="1.2.840.10045.4.3.4")
-# RFC4055
+# RFC 4055
 ID_RSASSA_PSS = ObjectIdentifier(value="1.2.840.113549.1.1.10")
 # RFC 4491
 ID_GOSTR3411_94_WITH_GOSTR3410_94 = ObjectIdentifier(value="1.2.643.2.2.4")
 ID_GOSTR3411_94_WITH_GOSTR3410_2001 = ObjectIdentifier(value="1.2.643.2.2.3")
-
+# RFC 3447
+ID_SHA256_WITH_RSA_ENCRYPTION = ObjectIdentifier(value="1.2.840.113549.1.1.11")
+ID_SHA384_WITH_RSA_ENCRYPTION = ObjectIdentifier(value="1.2.840.113549.1.1.12")
+ID_SHA512_WITH_RSA_ENCRYPTION = ObjectIdentifier(value="1.2.840.113549.1.1.13")
+ID_SHA224_WITH_RSA_ENCRYPTION = ObjectIdentifier(value="1.2.840.113549.1.1.14")
```

Comment:
The SHA224 OID is from RFC 4055 (it does not appear in RFC 3447), unlike the other three.